spk/synocli-file/Makefile

```diff
@@ -54,6 +55,6 @@ SPK_LINKS += /usr/local/bin/sfv-hash:bin/rhash
 SPK_LINKS += /usr/local/bin/tiger-hash:bin/rhash 
 SPK_LINKS += /usr/local/bin/tth-hash:bin/rhash 
 SPK_LINKS += /usr/local/bin/whirlpool-hash:bin/rhash 
-
+SPK_LINKS += /usr/local/bin/rar2fs:bin/rar2fs
```

Comment:
You need `SPK_LINKS` only when link and target have different names. This will create exactly the same link as the line above with `SPK_COMMANDS += bin/rar2fs`. 
Please remove this one.

Reply:
Right, it's been removed.